pystan/model.py

```diff
@@ -54,20 +53,28 @@ def load_module(module_name, module_path):
         return imp.load_module(module_name, *module_info)
 
 
-def _map_parallel(function, args, n_jobs):
+def _map_parallel(function, args, n_jobs, n_threads=None):
     """multiprocessing.Pool(processors=n_jobs).map with some error checking"""
     # Following the error checking found in joblib
     multiprocessing = int(os.environ.get('JOBLIB_MULTIPROCESSING', 1)) or None
     if multiprocessing:
         try:
             import multiprocessing
             import multiprocessing.pool
+            if n_jobs < 1:
```

Comment:
I think we calculate the number of jobs elsewhere.

Reply:
Yes, this was left here by an accident. I will remove this.